region/maridia/outer/Crab Tunnel.json

```diff
@@ -443,8 +451,16 @@
         {"resetRoom": {
           "nodes": [2]
         }},
-        {"refill": ["Energy", "PowerBomb"]}
-      ]
+        {"partialRefill": {
+          "type": "Energy",
+          "limit": 100
```

Comment:
The right door should be fine to be higher, it has a lot of crabs. Looks like top of crab hole is a full refill, this doesnt seem like it would be too much slower

Reply:
I mistook this room for `West Sand Hall Tunnel`